lib/ddtrace/span.rb

```diff
@@ -72,11 +73,13 @@ def initialize(tracer, name, options = {})
       @parent = nil
       @sampled = true
 
-      @start_time = nil # set by Tracer.start_span
-      @end_time = nil # set by Span.finish
-
       @allocation_count_start = now_allocations
       @allocation_count_finish = @allocation_count_start
+
+      @start_time = nil
```

Comment:
Worth having comments here on what the difference between start/end time and duration are?

Reply:
agreed, updated